pythonx/cm_sources/alchemist.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# For debugging, use this command to start neovim:
+#
+# NVIM_PYTHON_LOG_FILE=nvim.log NVIM_PYTHON_LOG_LEVEL=INFO nvim
+
+import os, sys
+PLUGIN_BASE_PATH = os.path.abspath("%s/../../../" % __file__)
+sys.path.insert(0, PLUGIN_BASE_PATH)
+from elixir_sense import ElixirSenseClient
+import re
+
+DEBUG = False
+
+from cm import register_source, getLogger, Base
+register_source(name='alchemist',
+                priority=8,
+                abbreviation='ex',
+                scopes=['elixir'],
+                cm_refresh_patterns=[r'\.'],)
+
+logger = getLogger(__name__)
+
+class Source(Base):
+    def __init__(self, nvim):
+        super(Source, self).__init__(nvim)
+        self.re_suggestions = re.compile(r'kind:(?P<kind>.*), word:(?P<word>.*), abbr:(?P<abbr>.*), menu:(?P<menu>.*), info:(?P<info>.*)$')
```

Comment:
I am not Python pro, but I think that `[^,]` instead of `.` in regex should behave better as it would require less backtracks.

Reply:
Just for the record I tried `[^,]` and it failed to match with 
```
kind:f, word:Enum.all?, abbr:all?(enumerable, fun \\ fn x -> x end), menu: Enum, info: @spec all?(t, (element -> as_boolean(term))) :: boolean<n>Returns true if the given `fun` evaluates to true on all of the items in the enumerable.
```

I'll take a look again to find a better regex.